unified-codes/apps/data-service/src/app/v2/types/property/Property.ts

```diff
@@ -0,0 +1,27 @@
+export enum EPropertyField {
+  Type = 'type',
+  Value = 'value',
+};
+
+export enum EPropertyType {
+  DrugInteraction = 'drug_interaction',
+}
+
+export interface IProperty {
+  type: EPropertyType;
+  // TODO: strongly type property values.
+  value: unknown;
+}
+
+// TODO: make Property abstract.
```

Comment:
No changes required - just wondering what the plan is here? Was it to implement specific property types for all the possible properties? (e.g. NZULM, ATC etc).

Reply:
Yup, that's the idea!

![noidea](https://user-images.githubusercontent.com/43223637/98892634-9a605200-2505-11eb-98c0-b94513cacd3d.gif)